agent/workers/poller.go

```diff
@@ -10,9 +10,9 @@ import (
 	"github.com/fluidtruck/deepcopy"
 	"github.com/kubeshop/tracetest/agent/client"
 	"github.com/kubeshop/tracetest/agent/proto"
-	"github.com/kubeshop/tracetest/agent/workers/datastores/connection"
 	"github.com/kubeshop/tracetest/server/datastore"
 	"github.com/kubeshop/tracetest/server/tracedb"
+	"github.com/kubeshop/tracetest/server/tracedb/connection"
```

Comment:
We are pointing everything to the server trace db logic, in the future we have to kill that package to use the one from the agent, the polling error was caused because of the errorTraceNotFound instance was different between server/tracedb and agent/datastores package